process.c

```diff
@@ -826,12 +817,12 @@ pst_to_s(VALUE st)
 
 /*
  *  call-seq:
- *     stat.inspect   -> string
+ *    inspect -> string
  *
- *  Override the inspection method.
+ *  Returns a string representation of +self+:
  *
  *    system("false")
- *    p $?.inspect #=> "#<Process::Status: pid 12861 exit 1>"
+ *    $? # => #<Process::Status: pid 1246759 exit 1>
```

Comment:
We should explicitly call `inspect` because it's only IRB that implicitly calls `inspect` for the output.

Reply:
Duh.  inspect is the method being docked.  Fixed.